chunky/src/java/se/llbit/chunky/renderer/scene/Sky.java

```diff
@@ -162,9 +162,24 @@ public static SkyMode get(String name) {
   /** Current sky rendering mode. */
   private SkyMode mode = SkyMode.DEFAULT;
 
+  /** Simulated skies. */
+  public final static List<SimulatedSky> skies = new ArrayList<>();
+
+  static {
+    skies.add(new PreethamSky());
+    skies.add(new NishitaSky());
+  }
+
+  /** Simulated sky mode. */
+  private int simulatedSkyMode = 0;
```

Comment:
Plugins could add new sky modes. Should we use the classname to identify them?

Reply:
I switched it to hold a `SimulatedSky` reference instead which should be more robust to additional sky modes (b3fb88d)